package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Shopify's javascript application utilities",
   "scripts": {
     "publish": "lerna publish",
-    "prebuild": "yarn run clean",
-    "build": "lerna run build",
+    "build": "yarn tsc --build packages",
     "lint": "yarn eslint '**/*.{ts,tsx}'",
     "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
+    "pretest": "yarn run build",
     "test": "NODE_ICU_DATA=node_modules/full-icu jest --maxWorkers=3 --coverage=false",
     "check": "lerna run check",
     "release": "lerna publish",
-    "clean": "rimraf './packages/*/dist/**/*.{js,d.ts}'",
+    "clean": "rimraf './packages/*/dist/**/*.{js,d.ts}' './packages/*/*.tsbuildinfo' './packages/*.tsbuildinfo'",
```

Comment:
I know `dist/src` is annoying, but it's unavoidable for most projects.  So I'd prefer standardising on that, even for projects that currently use `dist`.

Reply:
More importantly, how are you planning to have `tsbuildinfo` files cached in CI?  I'm still at the top of the PR, so I may be missing the magic that makes it happen.

Reply:
> So I'd prefer standardising on that, even for projects that currently use dist

I've updated all the monorepo tsconfigs to reflect this as sewing-kit does. (`'.'` as the `baseUrl` and `rootDir`). Also changed the number of imports from `./dist/foo` to `./dist/src/foo`.

Reply:
> how are you planning to have tsbuildinfo files cached in CI?

I tried getting Travis to only cache the `tsbuildinfo` files, but I couldn't find any docs or working examples of caching files. Only directories. How does sewing-kit cache `tsbuildinfo` files? 

In Travis, what if we move all the `tsbuildinfo`s to a `tmp_build_cache` directory of some sort after a build and dump them back into their respective `packages/*` paths before a subsequent build starts.